Automattic/jetpack
# Add new connect splash screen content.

Fixes #8900 and #8617 

This still needs some work:
- [x] URLs for white buttons needed.
- [x] Scrolling doesn't work because of the absolutely positioned element.
- [x] The 'not now' button doesn't do anything.
- [x] Paddings and margins seem to be off.

I was hoping I could get some help with this from the design team.

![screenshot from 2018-03-23 22-35-20](https://user-images.githubusercontent.com/374293/37849987-b83e4606-2eea-11e8-8248-e35e2d740535.png)


#### Changes proposed in this Pull Request:
* Adds new content, includes illustrations from #9097 
* Cleans up old unused styles.

#### Testing instructions:
* Deactivate Jetpack from the plugins menu.
* Activate Jetpack again, see the splash screen

#### Proposed changelog entry for your changes:
* Refreshed the connect splash screen to help new users learn more about Jetpack.
